src/libraries/Reserves.sol

```diff
@@ -3,40 +3,48 @@ pragma solidity ^0.8.20;
 
 import {Currency} from "../types/Currency.sol";
 import {CustomRevert} from "./CustomRevert.sol";
+import {IPoolManager} from "../interfaces/IPoolManager.sol";
 
 library Reserves {
```

Comment:
Ok heres a suggestion for naming. Open to suggestions though

- `library CurrencyReserves`
- `constant RESERVES_OF_SLOT` ✅ 
- `SYNC_SLOT` -> `CURRENCY_SLOT`
- `function checkSync` -> `function checkCurrencyNotSet`
- `Currency syncing` -> `Currency currencySet`
- `function getSyncedCurrency` -> `function getCurrency`
- `function getReserves` ✅ 
- `function setReserves` -> `function setCurrencyAndReserves`

Reply:
Thanks a lot for the suggestions!
I am wondering about removing sync out of some of the names... Does it not provide more clarity to keep Synced in the naming, Since all of it is so closely connected to the process of syncing?
I do agree with the fact that there is no clear line currently though, but I personally think it might help with readability by leaning into the other direction (I have included the changes I fully agree with for clarity) :

- `library Reserves` -> `library CurrencyReserves`
- `constant RESERVES_OF_SLOT` ✅
- `SYNC_SLOT` -> `CURRENCY_SLOT`
- `function checkSync` -> `function checkNotSynced`
- `Currency syncing` -> `Currency currencySynced`
- `function getSyncedCurrency` ✅
- `function getReserves` -> `getSyncedReserves`
- `function setReserves` -> `function syncCurrencyAndReserves`

Let me know what you think, happy to stick with your naming suggestions, if you like it more that way :)

Reply:
_-For documentation purposes-_
After an internal discussion, we decided on these namings:
- `library Reserves` -> `library CurrencyReserves`
- `constant RESERVES_OF_SLOT` ✅
- `SYNC_SLOT` -> `CURRENCY_SLOT`
- `function checkSync` -> `function requireNotSynced`
- `Currency syncing` -> `Currency currencySynced`
- `function getSyncedCurrency` ✅
- `function getReserves` -> `getSyncedReserves`
- `function setReserves` -> `function syncCurrencyAndReserves`